### ISSUE TITLE
Regular C is undefined / C is not a function errors in console

### ISSUE THREAD

**Issue by Porkepix** [ajbura/cinny#1570]
### Describe the bug

I can regularly find these errors in the console logs

```
[Error] Caught /sync error – TypeError: C.close is not a function. (In 'C.close()', 'C.close' is undefined) — index-eaa22fd4.js:57071:23887
TypeError: C.close is not a function. (In 'C.close()', 'C.close' is undefined) — index-eaa22fd4.js:57071:23887
	(anonymous function) (index-eaa22fd4.js:62:26356)
```

and
```
[Error] Failed to process events on room !foo:matrix.org: – TypeError: C.close is not a function. (In 'C.close()', 'C.close' is undefined) — index-eaa22fd4.js:57071:23887
TypeError: C.close is not a function. (In 'C.close()', 'C.close' is undefined) — index-eaa22fd4.js:57071:23887
	(anonymous function) (index-eaa22fd4.js:62:33114)
```

### Reproduction

I have no STR

### Expected behavior

These errors probably shouldn't happen (or be more clear)

### Platform and versions

```shell
Archlinux and Cinny desktop 3.2.1 (didn't open the issue on the desktop side as I can't tell if it's exclusive to that one).
```

### Additional context

I'm trying to track down an issue of some messages not received on Cinny while being received on other clients. After Cinny's restart message is here. These errors might be totally unrelated to that issue, but I've no real clue for that one for now, so I'm reporting what I can find.

**Comment by aceArt-GmbH** [comment]
Seems to be the same issue as https://github.com/cinnyapp/cinny-desktop/issues/252 which points to https://github.com/cinnyapp/cinny/blob/fcd7723f73f1e89f4ea9fabd6e0c03c4c8a4ffd4/src/client/state/Notifications.js#L299-L302

Can't reproduce in web, but in desktop/tauri.
Get notification, but read in other client.
![image](https://github.com/cinnyapp/cinny/assets/33117017/50de9ab1-2ed3-4c53-907c-0e2e5d31fcc3)
C has no `.close()`

**Comment by ajbura** [comment]
this has been fixed in v4.0.0
